.nvmrc

```diff
@@ -1 +1 @@
-v18
+v18.18
```

Comment:
👎 for this change, as it pins to a version with a security vulnerability

Yari should work with the latest v18 version.

Reply:
```suggestion
v18
```